test/metabase/public_sharing/api_test.clj

```diff
@@ -1984,3 +2053,12 @@
                                                     card-id
                                                     (mt/id :people :latitude)
                                                     (mt/id :people :longitude))))))))))
+
+;;; --------------------------------- POST /oembed ----------------------------------
+
+(deftest oembed-test
+  (testing "GET /oembed"
+    (testing "Cannot export when enable-public-sharing is false"
```

Comment:
Is this `testing` description right? If so do we need to explicitly set `mt/with-temporary-setting-values` to make sure someone who's changed settings on their local dev doesn't have issues?

Reply:
Thanks, the description made it there as a bad copy/paste, thanks for catching it. There is no logic in the oembed API around the enable-public-sharing setting so I just removed that testing line.